src/main/java/seedu/address/storage/ManageHrStorage.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.storage;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import seedu.address.commons.exceptions.DataLoadingException;
+import seedu.address.model.ManageHr;
```

Comment:
Is this line a required import?

Reply:
yes it's utilised in line 12's (of this same file) @link tag for javadocs. this was also the case previously for AddressBook3